.github/workflows/build-native.yml

```diff
@@ -55,7 +55,8 @@ jobs:
         if: runner.os == 'macOS'
         run: |
           xcodes installed
-          sudo xcodes select 15.4
+          sudo xcodes select 16.0
```

Comment:
We shouldn't be building with beta software - was this a test or is there an issue building with Xcode 15?

Reply:
Yes, that is just me trying a long shot. I'm still seeing a consistent native crash on iOS that I cannot reproduce locally. I should have put the PR back in draft.